utils/schema/usersSchema.js

```diff
@@ -4,14 +4,55 @@ const { config } = require('../../config');
 const schema = mongoose.Schema;
 
 const UsersSchema = new schema({
-  FirstName: {
+  documentID: {
+    type: Number,
+    unique: true,
+    trim: true,
+    required: 'Please Supply an identification number',
+  },
+  firstName: {
+    type: String,
+  },
+  lastName: {
+    type: String,
+  },
+  email: {
     type: String,
+    unique: true,
+    lowercase: true,
+    trim: true,
+    required: 'Please Supply an email address',
   },
-  LastName: {
+  contactNumber: {
+    type: Number,
+  },
+  username: {
+    type: String,
+    unique: true,
+    lowercase: true,
+    trim: true,
+    default: function () {
+      const first = this.firstName.split(' ').slice(0, 1).join(' ');
+      const last = this.lastName.split(' ').slice(0, 1).join(' ');
+      const digits = this.documentID.toString().slice(-4);
+      const firstLast = first + '.' + last + '.' + digits;
+      return firstLast;
+    },
+  },
+  password: {
     type: String,
   },
+  createdAt: {
+    type: Date,
+    default: Date.now,
+  },
+  modifiedAt: {
+    type: Date,
```

Comment:
`default: Date.now`

Reply:
> `default: Date.now`

This comment is about ? Sorry I dont understand the reference @hyfi06

Reply:
on create the createAt, and updeteAt have same value.